src/core/operations_problem.jl

```diff
@@ -470,6 +468,41 @@ function _build!(
     return
 end
 
+function get_variables_value(op_m::OperationsProblem)
+    results_dict = Dict{Symbol, DataFrames.DataFrame}()
+    for (k, v) in get_variables(op_m.psi_container)
+        results_dict[k] = axis_array_to_dataframe(v)
+    end
+    return results_dict
+end
+
+function get_dual_values(op_m::OperationsProblem; kwargs...)
+    cons = get(kwargs, :constraints_duals, nothing)
+    isnothing(cons) && return Dict{Symbol, Any}()
+    results_dict = Dict{Symbol, DataFrames.DataFrame}()
+    for c in cons
+        v = get_constraint(op_m.psi_container, c)
+        results_dict[c] = axis_array_to_dataframe(v)
+    end
+    return results_dict
+end
```

Comment:
This returns different types. Maybe the following is more readable?

```
    cons = get(kwargs, :constraints_duals, [])
    return Dict(c => axis_array_to_dataframe(get_constraint(op_m.psi_container, c)) for c in cons)
```

Reply:
@Lilyhanig consider this in your PR to this branch, please.